internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -19,56 +19,79 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"sigs.k8s.io/kubebuilder/v3/pkg/cli"
-	cfgv2 "sigs.k8s.io/kubebuilder/v3/pkg/config/v2"
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	golangv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v2"
+	golangv3 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v3"
 
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/bundle"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/cleanup"
-	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/completion"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/generate"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/olm"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/run"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/scorecard"
 	"github.com/operator-framework/operator-sdk/internal/flags"
+	"github.com/operator-framework/operator-sdk/internal/plugins"
 	ansiblev1 "github.com/operator-framework/operator-sdk/internal/plugins/ansible/v1"
-	golangv2 "github.com/operator-framework/operator-sdk/internal/plugins/golang/v2"
-	golangv3 "github.com/operator-framework/operator-sdk/internal/plugins/golang/v3"
+	envtestv1 "github.com/operator-framework/operator-sdk/internal/plugins/envtest/v1"
 	helmv1 "github.com/operator-framework/operator-sdk/internal/plugins/helm/v1"
+	manifestsv2 "github.com/operator-framework/operator-sdk/internal/plugins/manifests/v2"
+	scorecardv2 "github.com/operator-framework/operator-sdk/internal/plugins/scorecard/v2"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
 )
 
 var commands = []*cobra.Command{
 	bundle.NewCmd(),
 	cleanup.NewCmd(),
-	completion.NewCmd(),
 	generate.NewCmd(),
 	olm.NewCmd(),
 	run.NewCmd(),
 	scorecard.NewCmd(),
 }
 
 func Run() error {
-	cli, _ := GetPluginsCLIAndRoot()
-	return cli.Run()
+	c, _ := GetPluginsCLIAndRoot()
+	return c.Run()
 }
 
 // GetPluginsCLIAndRoot returns the plugins based CLI configured to use operator-sdk as the root command
 // This CLI can run kubebuilder commands and certain SDK specific commands that are aligned for
 // the kubebuilder project layout
 func GetPluginsCLIAndRoot() (cli.CLI, *cobra.Command) {
+	ansibleBundle, _ := plugin.NewBundle("ansible"+plugins.DefaultNameQualifier, plugin.Version{Number: 1},
+		ansiblev1.Plugin{},
+		manifestsv2.Plugin{},
+		scorecardv2.Plugin{},
+	)
+	gov2Bundle, _ := plugin.NewBundle(golangv2.Plugin{}.Name(), golangv2.Plugin{}.Version(),
```

Comment:
As we spoke about and just to register and for we do not forget that. IMO: 
- the bundle idea is great. I think it has good potential to let us solve other problems by combining the plugins and re-using it and I'd like to keep this implementation/possibility provided by KB. 
- however, I still think that we need to track an issue upstream for in the future we try to make teh api provide a way for we define what are the default plugins that should be scaffolded by the CLI without combining them. 
- Then, would be nice have a comment here with the todo for the task and with a short explanation over how it works, why and etc,.

More info: https://github.com/operator-framework/operator-sdk/pull/4581#discussion_r584703801

Reply:
It already accepts a chain of plugins as the default plugins.

Reply:
Just to clarifies. I loved the bundle plugin idea and it is very useful. imo; kb is now an api/lib which can be used for any tool/project then I think I am trying to think out other requirements possibilities that we can cover with.  I will close this one for not lead misunderstand for others.